prerendering.bs

```diff
@@ -658,6 +673,41 @@ The <dfn attribute for="PerformanceNavigationTiming">activationStart</dfn> gette
 
 1. Return a {{DOMHighResTimeStamp}} with a time value equal to the [=current document=]'s [=Document/activation start time=].
 
+<h3 id="client hint cache">Interaction with Client Hint Cache</h3>
+
+We need to ensure that the [=Accept-CH cache=], which is owned by user agent as a global storage, is not modified while prerendering, but is properly updated after activation. The following modifications ensure this:
+
+Each [=prerendering browsing context=] maintains its <dfn for="prerendering browsing context">prerender-scoped Accept-CH cache</dfn>, to persist the origin's opt-in to receive the indicated header fields.
+
+The [=prerendering browsing context/prerender-scoped Accept-CH cache=] is an [=ordered map=] of [=origins=] to [=client hints sets=].
+
+<div algorithm="add a new prerender Accept-CH cache entry">
+
+Similar to [=add a new Accept-CH cache entry=], to <dfn>add a new prerender Accept-CH cache entry</dfn> to the [=prerendering browsing context/prerender-scoped Accept-CH cache=],
```

Comment:
nit: I would split out the assertion about similarity into a separate informative note. I personally mildly prefer separating the steps from the line that defines the parameters (even if there's only one step), but this might be a matter of taste.

nit: another matter of taste, but trying to figure out whether the indexing syntax binds more tightly than the `'s` took me a second; splitting it into two steps seems to me to make it easier -- but it's probably fine not to, if you'd prefer

```
To <dfn>add a new prerender Accept-CH cache entry</dfn> given a [=prerendering browsing context=] |bc|, an [=origin=] |origin|, and a [=client hints set=] |hintSet|:

1. Let |cache| be |bc|'s [=prerendering browsing context/prerender-scoped Accept-CH cache=].
1. Set |cache|[|origin|] to |hintSet|.
```

Reply:
This is an example where splitting it into a note aligns with my sense of style :).

Reply:
Thanks!! Split this one and others!